src/main/kotlin/com/strumenta/kolasu/model/Processing.kt

```diff
@@ -61,14 +61,14 @@ private fun providesNodes(kclass: KClass<*>?): Boolean {
 }
 
 /**
- * @return can this class be considered an AST node?
+ * @return can [this] class be considered an AST node?
```

Comment:
why the square brackets?

Reply:
It's a link, check out https://kotlinlang.org/docs/reference/kotlin-doc.html